kratos/python_scripts/assign_vector_variable_process.py

```diff
@@ -35,17 +35,8 @@ def __init__(self, Model, settings ):
                 for i in range(3):
                     settings["constrained"][i].SetBool(is_fixed)
 
-
-        #detect "End" as a tag and replace it by a large number
-        if(settings.Has("interval")):
-            if(settings["interval"][1].IsString() ):
-                if(settings["interval"][1].GetString() == "End"):
```

Comment:
Why do you remove this, I don't think this is duplicated

Reply:
It is implemented in IntervalUtility and called though the assign_scalar-variable_process.py

Reply:
yep I see